partner/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+class Counselor(models.Model):
+    name = models.CharField(max_length=50)
+    gender = models.CharField(max_length=10)
+    level = models.ForeignKey('Level', on_delete=models.CASCADE)
+    counsel_count = models.IntegerField(default=0)
+    introduction = models.CharField(max_length=1000)
+    is_counsel_gt_150 = models.IntegerField(default=0)
```

Comment:
필드전체 = 기준으로 정렬 해주세요. 
```suggestion
    introduction      = models.CharField(max_length=1000)
    is_counsel_gt_150 = models.IntegerField(default=0)
```